connector/connect/common/src/main/protobuf/spark/connect/base.proto

```diff
@@ -375,6 +375,9 @@ message ExecutePlanResponse {
     // Response type informing if the stream is complete in reattachable execution.
     ResultComplete result_complete = 14;
 
+    // Response for command that creates ResourceProfile.
+    CreateResourceProfileCommandResult create_resource_profile_command_result = 20;
```

Comment:
Isn't the next ID 17 here?

Reply:
Hi @grundprinzip, Thx for reviewing, Changed it to 17. Could you help review it again?